spec/spec_helper.rb

```diff
@@ -9,6 +9,7 @@
 # Requires supporting ruby files with custom matchers and macros, etc,
 # in spec/support/ and its subdirectories.
 Dir[Rails.root.join("spec/support/**/*.rb")].each {|f| require f}
+Capybara.default_driver = :rack_test
```

Comment:
agreed, i always have bad luck with webkit and getting qt to play nicely together and selenium seems heavyweight.
there's really not a ton of js in the app right now anyway, the twitter feed integration is the biggest thing. we could probably test that separately anyway.